template.yaml

```diff
@@ -1924,6 +1923,11 @@ Resources:
             StartingPosition: TRIM_HORIZON
             BatchSize: 1
             Enabled: true
+            MaximumRetryAttempts: 3
+            BisectBatchOnFunctionError: true
+            DestinationConfig:
+              OnFailure:
+                Destination: !GetAtt DynamodbEventFanoutStreamRecordsDLQ.Arn
 
```

Comment:
Not setting max event age as events after batch scan and during cristin-import can be up to 12h old.